angular-ngrx-scss/src/app/repository/components/repo-header/repo-heading/repo-heading.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, Input, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-repo-heading',
+  templateUrl: './repo-heading.component.html',
+  styleUrls: ['./repo-heading.component.scss'],
+})
+export class RepositoryHeadingComponent implements OnInit {
+  @Input() owner = '';
+  @Input() name = '';
+
+  ownerPath = '';
+
+  ngOnInit(): void {
+    this.ownerPath = `/${this.owner}`;
```

Comment:
Same comment as for the repo-header.component.ts: I'd make this a getter or the owner a setter.